src/VisualStudio/Core/Def/Implementation/Snippets/AbstractSnippetExpansionClient.cs

```diff
@@ -393,7 +451,338 @@ public virtual bool TryInsertExpansion(int startPositionInSubjectBuffer, int end
                 iEndIndex = endIndex
             };
 
-            return expansion.InsertExpansion(textSpan, textSpan, this, LanguageServiceGuid, out ExpansionSession) == VSConstants.S_OK;
+            if (expansion.InsertExpansion(textSpan, textSpan, this, LanguageServiceGuid, out ExpansionSession) == VSConstants.S_OK)
+            {
+                // This expansion is not derived from a symbol
+                _symbols = default;
+                _symbol = null;
+                _arguments = _arguments.Clear();
+                return true;
+            }
+
+            if (!(SubjectBuffer.GetFeatureOnOffOption(CompletionOptions.EnableCallCompletionOnTabTab) ?? false))
+            {
+                // Full method call completion is not enabled
+                return false;
+            }
+
+            var symbols = ThreadingContext.JoinableTaskFactory.Run(() =>
+            {
+                var caretPosition = SubjectBuffer.CurrentSnapshot.GetPoint(endPositionInSubjectBuffer);
+                return GetSymbolsAsync(caretPosition, cancellationToken);
+            });
```

Comment:
using JTF here seems weird. just a few dozen lines above, we have:

```
SubjectBuffer.CurrentSnapshot.GetSpan(startPositionInSubjectBuffer, endPositionInSubjectBuffer - startPositionInSubjectBuffer),
```

so i don't ee why we need to end up using KTF to call to get the positoin again.

Reply:
The JTF usage is for `GetSymbolsAsync`. Can look into simplifying.

Reply:
⏱️ Simplified this by reusing the trigger span from earlier in the method